worksheet_A/TerminalHacking.cpp

```diff
@@ -40,7 +54,45 @@ int main()
 	}
 
 	// TODO: implement the rest of the game
+	int Likeness = 0;
+	int IncorrectGuesses = 0;
+	bool inlist;
+	std::string guess;
+	
+	while (IncorrectGuesses < 4)
+	{
+		inlist = false;
+		while (inlist == false)
+		{
+			std::cout << "Type word in UPPERCASE" << std::endl;
+			std::cin >> guess;
 
+			// loops through all the words in the word list
+			for each (std::string word in options)
+			{
+				// If the player guess is in the word list then it breaks out of the loop
```

Comment:
Comment does not correspond to code -- you actually don't break out of the loop.